packages/block-directory/src/plugins/installed-blocks-pre-publish-panel/index.js

```diff
@@ -11,9 +11,9 @@ import { blockDefault } from '@wordpress/icons';
 import CompactList from '../../components/compact-list';
 import { store as blockDirectoryStore } from '../../store';
 
-// We shouldn't import the edit-post package directly
-// because it would include the wp-edit-post in all pages loading the block-directory script.
-const { PluginPrePublishPanel } = window?.wp?.editPost ?? {};
+// We shouldn't import the editor package directly
+// because it would include the wp-editor in all pages loading the block-directory script.
+const { PluginPrePublishPanel } = window?.wp?.editor ?? {};
```

Comment:
Hm.. It seems I missed that in my PR. 

I guess it's fine to just import from  `@wordpress/editor` and remove this line and comments.